lib/assessment.rb

```diff
@@ -13,28 +15,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
```

Comment:
Correct, though this will only work in Ruby 2.0 and above; in older versions of Ruby, you must use `&` before the symbol:
`enum.reduce(&:+)`